src/lighteval/metrics/metrics_sample.py

```diff
@@ -161,10 +159,8 @@ def __init__(
                 Defaults to None if no normalization is applied.
             strip_strings (bool, optional): Whether to strip both reference and predictions. Defaults to False.
         """
-        if aggregation_function is None:
-            aggregation_function = max
-        self.aggregation_function = aggregation_function
 
+        self.aggregation_function = aggregation_function
```

Comment:
We still need to add a check to make sure the user did not accidentally pass None as aggregation_function (raising an exception should do)

Reply:
tbf if user disrespects the type signature that's on them! But okay I will keep it to be backward compatible